Framework/Crystal/src/TransformHKL.cpp

```diff
@@ -57,6 +57,10 @@ void TransformHKL::init() {
       "Specify 3x3 HKL transform matrix as a comma separated list of 9 "
       "numbers");
 
+  this->declareProperty("FindError", true,
```

Comment:
It is an agreed practice to add new properties at the end.
If a user script is using positional arguments instead of keyword arguments, this will break them.

Reply:
I think is makes for a better algorithm interface to put the new property next to existing related properties, for instance, new property `PixelWidth` next to existing property `PixelHeight`.
Users should be using positional arguments in their scripts 🙂

Reply:
I think you can still make them appear together in the dialogs if you assign them to the same property group.

Reply:
Unfortunately that's not the case. We recently had a PR where new properties were added at the end and assigned to an existing group. The GUI split the group in two with one part in the end, enclosing the new properties 😞

Reply:
This seems fine, the `NumIndexed` argument is an output so won't break any script using positional arguments.